File: fizzbuzz.m
for i = 1:100
    if mod(i,15) == 0
        disp("fizzbuzz")
    elseif mod(i,3) == 0
        disp("fizz")
    elseif mod(i,5) == 0
        disp("buzz")
    else
        disp(i)
    end
end

File: fizzbuzz.rs
fn main() {
    for i in 1..101 {
        match(i%3, i%5) {
            (0,0) => println!("fizzbuzz"),
            (0,_) => println!("fizz"),
            (_,0) => println!("buzz"),
            (_,_) => println!("{}",i),
        }
    }
}